firmware/integration/rusefi_config.txt

```diff
@@ -1337,9 +1343,9 @@ custom stepper_num_micro_steps_e 1 bits, U08,	@OFFSET@,	[0:3], @@stepper_num_mic
 	brain_pin_e mc33816_rstb;ResetB
 	brain_pin_e mc33816_driven
 
-	switch_input_pin_e brakePedalPin;Brake pedal switch
-	adc_channel_e cj125ua;lambda input
-	adc_channel_e cj125ur;heater input
+	switch_input_pin_e brakePedalPin;+Brake pedal switch
+	adc_channel_e cj125ua;+lambda input
```

Comment:
things like this don't need the `+` - those are truly only useful as code comments, not in-TS comments